CHANGELOG.md

```diff
@@ -48,6 +48,11 @@ Users/clients that do not make use of these endpoints will not be impacted.
   - Added optional `maxDownloadTime` field to `provider` schema
   - Added `max_download_time` column to PostgreSQL `providers` table
   - Updated `@cumulus/ingest/lock` to check expired locks based on `provider.maxDownloadTime`
+- **CUMULUS-3368**
+  - Added `allFilesPresent` field for `collection.meta` that defines whether or not all files
```

Comment:
Since `allFilesPresent` is used in our task, suggest to add it to https://github.com/nasa/cumulus/blob/master/packages/api/lib/schemas.js#L211

Reply:
Also, please add the following to collection and granule mapping: 
https://github.com/nasa/cumulus/blob/master/packages/es-client/config/mappings.json
"meta": {
        "type": "object",
        "dynamic": false
      },
There is a ticket [CUMULUS-3417](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3417) about mapping issue, and it can happen for collection record as well